addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiClientThingHandler.java

```diff
@@ -90,7 +93,7 @@ public void initialize() {
             return;
         }
 
-        UniFiControllerThingHandler controllerHandler = (UniFiControllerThingHandler) bridge.getHandler();
+        controllerHandler = (UniFiControllerThingHandler) bridge.getHandler();
```

Comment:
If you make `controllerHandler` a field also take care of the other places in this class that use the value of field such that the code become better. Let other places also make use of this field.

Reply:
Think I found the rest - and updated.